Organism.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace WorldProject;
+
+internal abstract class Organism
+{
+    public int Power { get; set; } = 0;
+    public int Init { get; set; } = 0;
+
+    public int X { get; set; } = 0;
+    public int Y { get; set; } = 0;
+
```

Comment:
it is necessary to create a constructor that will accept the coordinates of the organism